scripts/eurostat/health_determinants/common/EuroStat.py

```diff
@@ -0,0 +1,359 @@
+# Copyright 2022 Google LLC
```

Comment:
Pls use lower snake case for file names: `euro_stat.py`
https://screenshot.googleplex.com/6saUPe9UPh89fXd